aioredis/commands/streams.py

```diff
@@ -209,9 +209,9 @@ def xack(self, stream, group_name, id, *ids):
         """Acknowledge a message for a given consumer group"""
         return self.execute(b"XACK", stream, group_name, id, *ids)
 
-    def xdel(self, stream, id):
+    def xdel(self, stream, id, *ids):
         """Removes the specified entries(IDs) from a stream"""
-        return self.execute(b"XDEL", stream, id)
+        return self.execute(b'XDEL', stream, id, *ids)
```

Comment:
```suggestion
        return self.execute(b"XDEL", stream, *ids)
```

Following the above, just being concise. Plus, black linter only accepts "" in this scenario.